changes/2532-uriyyo.md

```diff
@@ -0,0 +1 @@
+Fix issue with config decl at class level.
```

Comment:
Do you think you could improve the text to make it more explicit please?

Reply:
Done